matlab_simulation/05-estimation/fastSLAM.m

```diff
@@ -0,0 +1,104 @@
+function[newPose,map,y,muParticleNew,covParticleNew,newParticleSet,meas_ind,newfeature,centroid_particles,w_new]...
+        =fastSLAM(oldPose,u,MotionNoise,MeasurementNoise,map,particleSet,muFeatOld,covFeatOld,w_new,rmax,thmax,timeInterval,totalParticles,newfeature)
```

Comment:
Add usage and description of each argument here. That will be helpful for newbies.